packages/react/src/components/Notification/index.js

```diff
@@ -5,53 +5,4 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import {
-  NotificationActionButton as NotificationActionButtonNext,
-  NotificationButton as NotificationButtonNext,
-  ToastNotification as ToastNotificationNext,
-  InlineNotification as InlineNotificationNext,
-  ActionableNotification as ActionableNotificationNext,
-} from './next/Notification';
-import {
-  NotificationActionButton as NotificationActionButtonClassic,
-  NotificationTextDetails as NotificationTextDetailsClassic,
-  NotificationButton as NotificationButtonClassic,
-  ToastNotification as ToastNotificationClassic,
-  InlineNotification as InlineNotificationClassic,
-} from './Notification';
-import { createComponentToggle } from '../../internal/ComponentToggle';
-
-export const NotificationActionButton = createComponentToggle({
-  name: 'NotificationActionButton',
-  next: NotificationActionButtonNext,
-  classic: NotificationActionButtonClassic,
-});
-
-export const NotificationTextDetails = createComponentToggle({
-  name: 'NotificationTextDetails',
-  classic: NotificationTextDetailsClassic,
-});
-
-export const NotificationButton = createComponentToggle({
-  name: 'NotificationButton',
-  next: NotificationButtonNext,
-  classic: NotificationButtonClassic,
-});
-
-export const ToastNotification = createComponentToggle({
-  name: 'ToastNotification',
-  next: ToastNotificationNext,
-  classic: ToastNotificationClassic,
-});
-
-export const InlineNotification = createComponentToggle({
-  name: 'InlineNotification',
-  next: InlineNotificationNext,
-  classic: InlineNotificationClassic,
-});
-
-export const ActionableNotification = createComponentToggle({
-  name: 'ActionableNotification',
-  next: ActionableNotificationNext,
-  classic: null,
-});
+export * from './Notification';
```

Comment:
One quick question, for the exports here can we explicitly name the exports instead of `export *`? In the past, this has helped unintended exports not sneak through 😅

Reply:
Yup! Thanks for the feedback